test/functional/issue_templates_settings_controller_test.rb

```diff
@@ -23,6 +23,12 @@ def setup
     Role.find(1).add_permission! :manage_issue_templates
 
     @project = Project.find(1)
+    Setting.text_formatting = 'textile'
+  end
+
+  def teardown
```

Comment:
This exact function is defined 3 times in different test. Maybe putting it in the common place (maybe test_helper.rb?) will avoid code diplications.

Reply:
That's right. I will revise as suggested.

Reply:
[feb25c0](https://github.com/agileware-jp/redmine_issue_templates/pull/59/commits/feb25c0bf85fe697df7686e7b579563a9099af90) 
Moved definition to test_helper.rb to avoid code duplication.